jba/src/visualization/pages/4_Inspections_stats.py

```diff
@@ -0,0 +1,166 @@
+import json
+from pathlib import Path
+from typing import List, Dict, Tuple
+
+import pandas as pd
+import plotly.express as px
+import streamlit as st
+from diff_viewer import diff_viewer
+
+from core.src.model.column_name import SubmissionColumns
+from core.src.model.report.hyperstyle_report import HyperstyleReport
+from core.src.utils.df_utils import read_df
+from jba.src.inspections.analysis import (
+    find_unique_inspections,
+    get_unique_inspections_stats,
+    get_inspection_fixing_examples,
+)
+from jba.src.models.edu_columns import EduColumnName
+
+INSPECTIONS_TO_IGNORE = ['KDocMissingDocumentation', 'UnusedSymbol']
```

Comment:
Maybe you could set up a config to ignore inspections? And put these inspections as the default ones